### ISSUE TITLE
Using @CreatedAt causes Sequelize to also create updatedAt column

### ISSUE THREAD

**Issue by interphx** [RobinBuschmann/sequelize-typescript#50]
Model:

```
@Table({tableName: 'user'})
class User extends Model<User> {
  ...
    @CreatedAt
    @Column({ field: 'created_at' })
    createdAt: Date;
  ...
}
```
Expected Sequelize output:

> Executing (default): CREATE TABLE IF NOT EXISTS `user` (... `created_at` DATETIME ...) ...

Actual Sequelize output:

> Executing (default): CREATE TABLE IF NOT EXISTS `user` ( ... `created_at` DATETIME, `updatedAt` DATETIME NOT NULL ...) ...

This also makes methods like `findAll` unusable if my tables were created manually and do not contain `updatedAt` field.

**Comment by RobinBuschmann** [comment]
@interphx thanks for reporting.

**Comment by RobinBuschmann** [comment]
@interphx As a workaround you could use `@Table({ updatedAt: false })` temporarily.

**Comment by RobinBuschmann** [comment]
sequelize requires `timestamp: true` to be set, when using custom named `updatedAt` or `createdAt`. So, when using `@CreatedAt creationDate: Date;`, the options `timestamp: true` and `createdAt: 'creationDate'` are set internally. This in turn causes the `updatedAt` timestamp to be created too. The only way to prevent this, is setting `updatedAt: false`. But doing this automatically on the usage of `@CreatedAt` is not a good idea from my point of view. This could lead to unexpected behaviour.

In conclusion I recommend deactivating the timestamp fields explicitly if they should not appear on the table, when using `@CreatedAt, @UpdatedAt, @DeletedAt`.  

Regarding the current issue: `@Table({ updatedAt: false })` is still the way to go.

**Comment by BruceHem** [comment]
Yes I have this issue too, with only @DeletedAt, it also adds createdAt and updatedAt

**Comment by RobinBuschmann** [comment]
@BruceHem So then you need to deactivate them explicitly like: `@Table({ createdAt: false, updatedAt: false })`
